static/js/show-task.js

```diff
@@ -39,20 +39,46 @@ function saveTask() {
     }});
 }
 
+function rememberState() {
+        // Finding how many rows are being displayed so that we can make it same after refresh
+        var rowsPerPageDropdownIndex = $('.page-list .dropdown-menu .dropdown-item.active').index();
```

Comment:
By the way, now that I've looked at the docs for bootstrap-table, it sounds like this could be

```javascript
var rowsPerPageDropdownIndex = $('tasks_table').bootstrapTable('getOptions').pageSize;
var rowsPerPageIndex = $('tasks_table').bootstrapTAble('getOptions').pageNumber;
```

Skimming the code for bootstrap-table suggests that these values are updated dynamically as we switch pages ([start here](https://github.com/wenzhixin/bootstrap-table/blob/f73da419cfa244312710bb975d1857ff2ef544a7/dist/bootstrap-table.js#L1623)), but for some reason when I actually try the code above in our tool it only ever returns the initial values (10 and 1).

I'm OK with the current state of the code, but thought I'd point this out as you might spot something I'm missing. If we can rely on the public interface of the library to get these values instead of relying on class names (which are probably an implementation detail), that would be better. If you can't get it to work either, just leave a TODO please.

Reply:
This didn't work for me either! I'll give it one more shot and If I'm still unable to do it, I'll leave a TODO.